editor/src/clj/editor/app_view.clj

```diff
@@ -869,13 +869,13 @@
           {:title "Launch Failed"
            :icon :icon/triangle-error
            :header {:fx/type fx.v-box/lifecycle
-                    :children [{:fx/type fxui/label
+                    :children [{:fx/type fxui/legacy-label
```

Comment:
There was a label component, but now there is another, which is also used in editor scripts. I had to rename the old one...